index.html

```diff
@@ -1,13 +1,32 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title>JS Onboard Project</title>
+	<link rel="stylesheet" href="stylesheet.css">
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	<script type="text/javascript" charset="utf-8" src="app.js"></script>
+	<script type="text/javascript" charset="utf-8" src="api-service.js"></script>
+	<script type="text/javascript" charset="utf-8" src="grid-template.js"></script>
```

Comment:
You want to order these in the order that they are needed. Pretty sure `app.js` needs `api-service.js`, so it will have to be loaded first. and so on and so forth.